src/main/java/com/offz/spigot/staminaclimb/Climbing/ClimbBehaviour.java

```diff
@@ -138,9 +139,39 @@ public void onRightClick(PlayerInteractEvent e) {
         Location L1 = p.getLocation();
 
         Vector v = p.getVelocity();
+
+        //todo make pick improve stamina and speed?
+
+        if (allowClimb(p) && rightClicked(e) && cooldownComplete(uuid) && (!p.isSneaking() || v.getY() < -0.5) && !isClimbing.containsKey(uuid) && p.getInventory().getItemInMainHand().getType().equals(Material.IRON_PICKAXE) && p.getInventory().getItemInMainHand().getItemMeta().getDisplayName().equals(ChatColor.LIGHT_PURPLE + "Climbing Axe")) {
```

Comment:
Use Geary to find the item you're looking for, having to manually do the display name check every time isn't clean!
Ask @Derongan how that should be done, I could also explain how to add geary as a dependency in the build.gradle